pkg/tempopb/tempo.proto

```diff
@@ -26,6 +26,10 @@ service Querier {
   rpc SearchTagValuesV2(SearchTagValuesRequest) returns (SearchTagValuesV2Response) {};
 }
 
+service StreamingQuerier {
+  rpc Search(SearchRequest) returns (stream SearchResponse);
```

Comment:
Should we also stream the requests?

Reply:
I considered that, but it's more complex and given the nature of Tempo queries I think that a one way stream is sufficient.